recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
+  sha256: c54143587f74a120f77987ea9a2d011e1c090709f07e3678f25aa1ffb93a6777
+
+build:
+  number: 0
+  skip: true  # [not win]
+  
+test:
+  commands:
+    - IF NOT EXIST %SCRIPTS%\setup_x64.bat exit 1
+    - IF NOT EXIST %SCRIPTS%\x64 exit 1
+
+about:
+  summary: Workaround to help CMake find the appropriate x64 compiler on VS 2008 Express with VS compiler for python on top.
```

Comment:
Please pin to v0.4.8 as I am deleting that file in condaci. As a note, you don't really need it for MSVC - it only matters with all the Intel architectures (Itanium etc). You just need to copy the [one file](https://github.com/menpo/condaci/blob/98d76b2089d494433ac2e3abd920123088a95a6d/condaci.py#L314).